zluda_blaslt/src/impl.rs

```diff
@@ -18,6 +29,84 @@ impl ZludaObject for Handle {
 
 from_cuda_object!(Handle);
 
+pub struct MatmulDesc {
```

Comment:
This whole wrapping of hipblasLtMatmulDesc_t, hipblasLtMatmulPreference_t and hipblasLtMatrixLayout_t seems unnecessary to me, if we wrapped those types with some extra data, but as it is now it's better to map them 1:1